Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -243,6 +243,20 @@ theorem lift_quot_mk {φ : G →* M} (HN : N ≤ φ.ker) (g : G) :
     lift N φ HN (Quot.mk _ g : Q) = φ g :=
   rfl
 
+@[to_additive]
+theorem lift_surjective_of_surjective (φ : G →* M) (hφ : Function.Surjective φ) (HN : N ≤ φ.ker) :
```

Comment:
Could you please add a `Quotient.lift'` version? See also `Function.Surjective.of_comp`.

Reply:
I added `Quotient.lift_surjective_iff` and `Quotient.lift_surjective` using [Quot.surjective_lift](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Quot.html#Quot.surjective_lift). I thought it would be easier to have both the `iff` version and the implication versions. I was not sure for the name whether to use something like `Quot.surjective_lift` and to use the general naming rule for surjective lemmas. I went with the general naming rule and put `surjective` at the end. Maybe the solution is to change `Quot.surjective_lift` to `Quot.lift_surjective_iff`...

Reply:
I golfed it by generalizing `comap_ker` instead. We should generalize the whole file to use `MulOneClass` or `Monoid` for the codomain.